data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -617,6 +617,46 @@
         {CLEAR_VARIABLE delfador_moves}
     [/event]
 
+    # Introductory dialog to give the player a hint that she may lose units to
+    # {EXPAND_LAVA}.
+    [event]
+        name=turn 2
+        # The noise was played by {EXPAND_LAVA}.
+        [message]
+            id=Konrad
+            message=_ "That noise was made by no troll! Can you tell its source?"
+        [/message]
+        [if]
+            [have_unit]
+                race=dwarf
+            [/have_unit]
+            [then]
+                [message]
+                    race=dwarf
+                    message=_ "Lord Konrad, yon rumble was made by nae enemy o’ us. ’Tis the echoes of the very bedrock witherin’ and crumbling beneath us that we ’eard. Are ye feeling naught heat and loess redoubling around us? I beseech that we not tarry ’ere any more than we ha’ to, lest the forming abysses claim us."
```

Comment:
Whoa, there's a word I'd never actually heard before. Should we add a "po" note giving the meaning of "loess" or should we assume translators have a dictionary handy?

Also, the phrasing of that sentence doesn't seem right - "naught" is a noun, generally meaning "nothing". Note that if you substitute "nothing" into the sentence, it doesn't make sense: *"Are ye feeling nothing heat and loess redoubling around us?". Did you mean to use "not"? If so it'd be spelled with an O, though I think that's also a pretty unusual usage (but maybe it's more common in some dialects). If "not" is what you meant, an alternative might be "Are ye nae feeling...", though that doesn't fully solve the issue - according to Wiktionary, "loess" means "sediment", so what does it mean for that to be redoubling around you? And also, _feeling_ it redouble (rather than seeing).

Reply:
> Whoa, there's a word I'd never actually heard before. Should we add a "po" note giving the meaning of "loess" or should we assume translators have a dictionary handy?

Well, I assumed dwarves had as many words for geological features of caves as Eskimos proverbially do for frozen water. :grinning: It's not an archaic word but it is very rare, so let's add a po comment to save translators time.

I'm not sure "loess" is the right word, though. As you say, "loess" means "sediment", but the meaning I was going for was "airborne dust and lint" from the collapsed walls. Got a better word?

> Also, the phrasing of that sentence doesn't seem right - "naught" is a noun, generally meaning "nothing".

Yeah, I wasn't sure about that part. I checked HttT and THoT and saw that dwarves use "naught" a lot so I tried to use an archaic form of negation in that sentence as well. Let's try to find an archaic structure then? We don't have to use the word "naught".

The meaning I was going for was "Aren't you feeling heat and [sensing] [airborne dust and lint]?". Then I flipped the negation over to "Are you sensing no heat and [airborne dust and lint]?" and (incorrectly) substituted "naught". I think I'm out of my depth on your suggestions to use "nought" and "nae". Where would "nae" be placed, "Are ye nae feeling heat" or "Are ye feeling nae heat"?

Reply:
I'm not sure about "nought". I went for "nae" and "the air, sowled by ashes". Wiktionary gives "sullow" and "sully" as synonyms but "sowl" felt closest to the dialects' pronunciation patterns.

More options: fume, miasma, effluvium.

Reply:
Just noticed this - fume does sound very workable, and haze and miasma aren't bad either.

Reply:
Indeed it is. In fact, the story already uses "fume" to describe the way taken from S16 to S17. In the dialog we should use more intense language, I think, now that the party is closer to the Sceptre and the floor is getting engulfed by lava. "Fumes from cracks in the floor" doesn't seem to cut it any more; "haze" and "ashes" feel more appropriate.

I think the use of "ashes" is good as it is. We could add "haze" or "miasma" if we have an opportunity to. Hmm, maybe let's go over the Bitter Swamp scenario to see if any of these words can be used there.